modules/aws-http4s/src/smithy4s/aws/AwsClient.scala

```diff
@@ -138,8 +139,10 @@ object AwsClient {
 
     def build[F[_]: Async: Compression](
         awsEnv: AwsEnvironment[F]
-    ): service.Impl[F] =
+    ): service.Impl[F] = {
+      implicit val hashingF: Hashing[F] = Hashing.forSync[F]
```

Comment:
didn't bother passing it from the outside and doing a deprecation on this `build` signature, but I'm open to suggestions